x/oracle/types/key_native_token.go

```diff
@@ -0,0 +1,50 @@
+package types
+
+import "strings"
+
+const (
+	// NativeTokenKeyPrefix is the prefix to retrieve all NativeToken
+	NativeTokenKeyPrefix                 = "NativeToken/"
+	NativeTokenPriceKeyPrefix            = NativeTokenKeyPrefix + "price/value/"
+	NativeTokenStakerInfoKeyPrefix       = NativeTokenKeyPrefix + "stakerInfo/value/"
+	NativeTokenOperatorInfoKeyPrefix     = NativeTokenKeyPrefix + "operatorInfo/value/"
+	NativeTokenStakerListKeyPrefix       = NativeTokenKeyPrefix + "stakerList/value/"
+	NativeTokenStakerDelegationKeyPrefix = NativeTokenKeyPrefix + "stakerDelegation/value/"
+
+// NativeTokenValidatorListKeyPrefix    = NativeTokenKeyPrefix + "validatorList/value"
+)
+
+func NativeTokenStakerDelegationKey(assetID, stakerAddr string) []byte {
+	assetID = strings.Join([]string{assetID, stakerAddr}, "/")
+	return append([]byte(NativeTokenStakerDelegationKeyPrefix), []byte(assetID)...)
+}
+
+func NativeTokenStakerListKey(assetID string) []byte {
```

Comment:
If some function is frequently called on a performance-critical path, consider using sync.Pool to reuse bytes.Buffer

Reply:
not frequently called

Reply:
not frequently